### DISCUSSION TITLE
Go to diagnostic across files

### DISCUSSION THREAD

**Discussion by 0xBradock** [helix-editor/helix#12122] [Q&A] [answered]
Hi, I am new to helix.

Let's suppose I am working with a strongly typed language.
When I make a modification in one file I brake the code, usually, in many other files.
I want to be able to go directly to the places my code broke.

Using the `]d Goto next diagnostic` only navigates within a file.

Is there a way to navigate across files to the next place I have a lsp error, kind of `gl Goto next lsp error|warning`.
Or, either, `<space>l Open lsp messages` then have a list of lsp warnings|errors along with the file name.

Thanks,

**Comment by kirawi** [comment, marked as answer]
`space-D`

**Reply by 0xBradock** [reply]
I missed that. Thanks
